Makefile

```diff
@@ -20,6 +20,9 @@ IMAGE_REPOSITORY    := $(REGISTRY)/etcd-druid
 IMAGE_TAG           := $(VERSION)
 BUILD_DIR           := build
 
+# TODO: Remove this as soon as vendored to new gardener/gardener version.
```

Comment:
Don't you want to put your name with TODO? Like TODO(timuthy) ? So that, in future if anybody wants to work on this TODO, they can connect you for context.

Reply:
Done. PTAL 🙂